docs/image/image.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CoolWaterUI</title>
+    <link rel="stylesheet" href="image.css">
+</head>
+<body>
+    <div class="sidebar-and-main-container">
+        <aside class="sidebar">
+
+            <div class="docs-navbrand-container"><a href="/"><div class="navbar-brand">CoolWaterUI</div></a></div>
```

Comment:
Here there's no need to put your brand name inside div <a> tag is enough. You can remove that div if there's no valid reason for it.